tests/src/RunBenchmarks/RunBenchmarks.cmd

```diff
@@ -0,0 +1,20 @@
+setlocal
+
+set BENCHMARK_ROOT_DIR=\git\repos\DotNet\coreclr\bin\tests\Windows_NT.x64.Release\JIT\Performance\CodeQuality
+set BENCHMARK_HOME=\git\repos\DotNet\coreclr\tests\src\RunBenchmarks
+set BENCHMARK_HOST=%BENCHMARK_HOME%\bin\Debug\dnxcore50\RunBenchmarks.exe
+set BENCHMARK_HOST=%BENCHMARK_HOME%\bin\Debug\desktop\RunBenchmarks.exe
+set BENCHMARK_RUNNER=-runner CoreRun.exe
+
+start DHandler.exe
```

Comment:
`DHandler` is super-annoying, I'd say we should not bother. There's less likelihood of GUI popups on coreclr anyways.